src/client/aureliaCLICommands.ts

```diff
@@ -12,6 +12,7 @@ export default class AureliaCliCommands {
       vscode.commands.registerCommand('extension.auGenerate', () => this.auGenerate(outputChannel)),
       vscode.commands.registerCommand('extension.auTest', () => this.runCommand(['test'], outputChannel, false)),
       vscode.commands.registerCommand('extension.auBuild', () => this.auBuild(outputChannel)));
```

Comment:
I just looked over my code after going through the hello world vscode extension sample and noticed that I needed to update the package.json to expose the new command... in two places. Tested and confirmed to run auRun >> au run.